pkgs/misc/cups/drivers/xerox6280/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, rpm, cpio} :
+stdenv.mkDerivation {
+  pname = "Xerox-Phaser-6280";
+  version = "1.0-1";
```

From the `meta.downloadPage`:

> Version: 1.0.0.0

```suggestion
  version = "1.0.0.0";
```

Within Nixpkgs we use the upstream version numbering scheme.

---

Not sure what happened there. Thought I copy pasted it from upstream. Fixed

---

Not sure what happened there. Thought I copy pasted it from upstream. Fixed